packages/api/cms-api/src/blocks/table.block.ts

```diff
@@ -0,0 +1,138 @@
+import { BlockData, BlockField, BlockInput, createBlock, inputToData } from "@comet/blocks-api";
+import { plainToInstance, Type } from "class-transformer";
+import { IsArray, IsBoolean, IsEnum, IsNumber, IsString } from "class-validator";
+
+enum ColumnSize {
+    extraSmall = "extraSmall",
+    small = "small",
+    standard = "standard",
+    large = "large",
+    extraLarge = "extraLarge",
+}
+
+class TableBlockColumnData extends BlockData {
+    @BlockField()
+    id: string;
+
+    @BlockField()
+    position: number;
+
+    @BlockField({ type: "enum", enum: ColumnSize })
+    size: ColumnSize;
```

Comment:
Should this be configurable?

Reply:
Do you mean the values or the setting itself? 
I think the 5 values already defined should be sufficient.

Reply:
The setting itself. As an editor I'd except that the table would fit the content automatically.

Reply:
> As an editor I'd except that the table would fit the content automatically.

Yes, it should and will. 

I don't think showing an optional setting in the admin hurts but we would always have to implement this behavior in the site block then 🤔 

Will discuss this with UX, if we ever explicitly don't want admins to be able to control the column size.

Reply:
As discussed with UX, there is no need to make this configurable.